### ISSUE TITLE
Failed to use build script

### ISSUE THREAD

**Issue by cool2528** [libvips/build-win64-mxe#32]
[download]    llvm-project-12.0.1.src.tar.xz
MXE Warning! Downloading llvm from second URL.
MXE Warning! Downloading llvm from backup.

Download failed or wrong checksum of package llvm!
------------------------------------------------------------
2021-07-29 08:41:43 ERROR 404: Not Found.
--2021-07-29 08:41:43--  http://d1yihgixbnrglp.cloudfront.net/llvm%2Dproject%2D12%2E0%2E1%2Esrc%2Etar%2Exz_129cb25cd13677aad951ce5c2deb0fe4afc1e9d98950f53b51bdcfb5a73afa0e
Reusing existing connection to d1yihgixbnrglp.cloudfront.net:80.
HTTP request sent, awaiting response... 404 Not Found
2021-07-29 08:41:44 ERROR 404: Not Found.
Download failed!
------------------------------------------------------------
[log]      /data/mxe/log/llvm-download

**Comment by kleisauke** [comment]
Looks like a temporary network failure. If restarting the build doesn't work, you can try downloading `llvm-project-12.0.1.src.tar.xz` manually from here:
https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.1/llvm-project-12.0.1.src.tar.xz

And place it in `build/mxe/pkg` to let the build script use that tarball instead of trying to download it.

Note that most people do not need to build the binaries themselves, as pre-built binaries can be downloaded from here:
https://github.com/libvips/build-win64-mxe/releases

(In addition, the first time build will take a long time, because LLVM and Rust must be built from source)

**Comment by cool2528** [comment]
First of all, thank you for your reply, the reason I compiled it was to test if the released version was built with the mozjpeg library by default, but I heard that mozjpeg is 2~3 times slower than the libjpeg-turbo library, so I needed to test if it was faster than your released version if I didn't use the mozjpeg library to build it. Is it? But unfortunately I'm in China and the firewall probably blocks many third-party open source library addresses so I can't build properly
So I'd like to know if the version you released using the mozjpeg library will be slower than the libjpeg-turbo library?
